bin/sts-assume-role.sh

```diff
@@ -13,7 +13,7 @@ region="$2"
 temp_role=$(aws sts assume-role \
                     --role-arn "${role_arn}" \
                     --role-session-name "concourse-task" \
-                    --duration 1800)
+                    --duration ${3:=1800})
```

Comment:
That's not what I mean. 

What I meant was keep the duration variable where it was but handle the default replacement there between lines 12 and 13. 

So instead of doing the default replacement when you call `aws sts` you do the default replacement when you read the input arguments.